version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.21.0-SNAPSHOT"
+version in ThisBuild := "0.21.0"
```

Comment:
I guess we can wait for:

* https://github.com/higherkindness/mu-scala/pull/772
* https://github.com/higherkindness/mu-scala/pull/770
* https://github.com/higherkindness/mu-scala/pull/768

Before releasing the next version?

Reply:
I'd say yes

Reply:
Yes, I would like to get the mu sbt plugin change out asap as the documentation has already been updated.

Reply:
I've just rolled back to the latest release version so we are not in a rush ;)

Reply:
Cool, will close this PR.